pkg/scaffold/scaffold.go

```diff
@@ -0,0 +1,179 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Modified from github.com/kubernetes-sigs/controller-tools/pkg/scaffold/scaffold.go
+
+package scaffold
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/operator-framework/operator-sdk/pkg/scaffold/input"
+	"github.com/operator-framework/operator-sdk/pkg/util"
+	"golang.org/x/tools/imports"
+)
+
+// Scaffold writes Templates to scaffold new files
+type Scaffold struct {
+	// Repo is the go project package
+	Repo string
+
+	// ProjectPath is the relative path to the project root
+	ProjectPath string
```

Comment:
Can you clarify what `ProjectPath` is? Here it says it's the relative path to the project root.
But above in the commands we're doing:
```Go
	absProjectPath := cmdutil.MustGetwd()
	cfg := &input.Config{
		Repo:        cmdutil.MustInProjectRoot(),
		ProjectPath: absProjectPath,
	}
```
Which makes it seem that its the absolute path to the project root.